app/models/patron_request.rb

```diff
@@ -45,6 +45,8 @@ def pickup_service_point
 
   # FOLIO
   def pickup_destinations
+    return Settings.allowed_visitor_pickups if patron.visitor_patron?
```

Comment:
This still needs to respect the location_restricted_service_point_codes, too, right? (e.g. PAGE-EA can only be picked up in EAST-ASIA)

Reply:
Yup. Thanks.